src/fiobncci.js

```diff
@@ -0,0 +1,12 @@
+function fiobnacci ( number ) {
+  var fibArray = [0, 1]
+  if ( number <= 0 ) return undefined;
+  if ( number === 1 ) return fibArray = [0]
```

Comment:
Since you are returning the array, there is no need to set it to the variable `fibArray` here. Same thing in the next line too